metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -294,7 +294,14 @@ def get_identifier(
     ) -> str:
         assert inspector
         project_id = self._get_project_id(inspector)
-        return f"{project_id}.{schema}.{entity}"
+        trimmed_table_name = (
+            BigQueryTableRef.from_spec_obj(
+                {"projectId": "project_id", "datasetId": schema, "tableId": entity}
```

Comment:
it doesn't affect the correctness of the code, but you are setting `projectId` to the string `"project_id"` instead of the variable `project_id` here.